### DISCUSSION TITLE
Listeners and Bandwidth, Beginner Question

### DISCUSSION THREAD

**Discussion by zandergordan** [AzuraCast/AzuraCast#7622] [General]
Hi, I want to set my expectations regarding performance limits of my new AzuraCast installation. It is running on an old Windows 10 PC via docker, on my home network. According to a random speed test I just did on the internet my upload speed is about 10 mbps, and I am streaming at 192kbps. Is the math as simple as 10,000/192 ~= 50 users is how many listeners I can have before I start encountering performance/bandwidth issues? 50 simultaneous listeners would be substantial success for me so, if so, I am not worried about running into that limit anytime soon, and if I ever get there I can look into other things like the AzuraRelay and/or VPS providers for expanding service.

Thanks for any insight.

**Comment by BusterNeece** [comment]
@zandergordan Basically, if AzuraCast is running well already with 0 listeners, but the AutoDJ is running and shuffling tracks and broadcasting to all the mount points you want, then the only limit after that is your upstream bandwidth, so the calculation really is pretty much as simple as you described.

There's a (relatively) constant load from just the inner workings of AzuraCast itself (the database, synchronized tasks, etc), and then the AutoDJ (Liquidsoap) imposes a very predictable CPU load for every signal it has to transcode to (i.e. every mount point it's broadcasting to). Icecast and/or Nginx serving HLS are built intentionally to scale to huge numbers with very little impact on resources, so once your station's up and running reliably, it's likely to continue doing so with a large listener count.
